site/_scss/layouts/devsite-theme.scss

```diff
@@ -1,9 +1,877 @@
+  --devsite-item-gap: #{$CARD_GAP};
+  --devsite-item-text-padding: 0;
+  --devsite-label-font: #{$LABEL_FONT};
+  --devsite-label-letter-spacing: #{$LABEL_LETTER_SPACING};
+  --devsite-primary-font-family: #{$TEXT_PRIMARY_FONT_FAMILY};
+  --devsite-primary-text-color: #{$TEXT_PRIMARY};
+  --devsite-text-color: #{$TEXT_PRIMARY};
+  --link-color: #{$TEXT_PRIMARY};
+  --link-rgb-background: #{$BUTTON_BG};
+  --link-visited-color: #{$TEXT_PRIMARY};
+
+
+  @media (max-width: 591px) {
+    --devsite-button-padding: #{$BUTTON_PADDING_MOBILE};
+    --devsite-buttons-margin: #{$BUTTON_MARGIN_MOBILE};
+    --devsite-landing-group-padding: 0;
+    --devsite-landing-row-padding: #{$ROW_PADDING_MOBILE};
+
+  }
+
+  .material-icons {
+    background-color: $CHROME_BLUE;
+    border-radius: 100%;
+    color: #fff;
+    display: inline;
+    padding: 0.46875rem;
+    position: relative;
+    top: 0.5rem;
+  }
+
+  .dcc-blog-card {
+    --devsite-buttons-margin: 0.5rem 0 0 -0.75rem;
+    --devsite-item-gap: 0;
+
+    padding: 5rem 2.125rem 3rem;
+
+    span {
+      position: absolute;
+      top: 2.25rem;
+    }
+
+    img {
+      border-radius: #{$CARD_BORDER_RADIUS};
+    }
+
+    &[description-position=left] {
+      align-items: flex-start;
+    }
+
+    &[description-position=bottom] {
+      padding: 5rem 1.125rem 3rem;
+
+      figure{
+        margin-bottom: 1rem;
+        padding: 0 1rem;
+      }
+    }
+
+    @media (max-width: 1280px) {
+      &[description-position=left] {
+        flex-direction: row-reverse;
+      }
+    }
+
+    @media (max-width: 840px) {
+      padding-left: 1rem;
+
+      figure {
+        margin-bottom: 1.5rem;
+      }
+
+
+
+      &[description-position=left],
+      &[description-position=right] {
+        align-items: center;
+        justify-content: space-between;
+        flex-direction: row;
+      }
+    }
+
+    @media (max-width: 591px) {
+      &[description-position=left] figure img,
+      &[description-position=right] figure img,
+      figure img {
+        height: 13.75rem;
+        object-fit: cover;
+        object-position: center;
+      }
+    }
+  }
+
+  .dcc-services-card {
+    --devsite-button-border-radius: 0;
+    --devsite-button-padding: 0;
+    --devsite-buttons-margin: auto;
+    --devsite-buttons-padding: 1rem 0 0 0;
+    --devsite-h2-margin: 1.6875rem auto auto;
+    --devsite-item-gap: 0;
+    --devsite-item-media-container-width: 11rem;
+    --devsite-item-media-margin: 0 auto;
+
+    margin-bottom: 2rem;
+    padding: 2.8rem 2.5rem;
+    
+    @media (max-width: 840px) {
+      margin-bottom: 0;
+    }
+
+    h3 {
+      max-width: 90%;
+      min-height: 8.75rem;
+      text-align: center;
+
+      @media (max-width: 840px) {
+        min-height: 6rem;
+      }
+
+      @media (max-width: 591px) {
+        min-height: auto;
+      }
+    }
+
+    h3 + div {
+      align-items: start;
+      flex-direction: column;
+      max-width: 12.5rem;
+    }
+
+    .button::before {
```

Comment:
Please make it a universal class, that can be also used outside of services card (like .dcc-icon-button). Adapt dcc-web-app-features-link calls below as well.

Reply:
@matthiasrohmer  I think you might reviewed the old file, we added `.dcc-icon-button` for each link and added style to this selector a few days back. Please see the [latest file here](https://github.com/GoogleChrome/developer.chrome.com/blob/4956d82e2973630b36fd268f5e720413ce6baa7c/site/_scss/layouts/devsite-theme.scss. ). Thank you 😊